src/test/deploy/functions/cel.spec.ts

```diff
@@ -0,0 +1,530 @@
+        resolveExpression("boolean", "{{ 22 == 11 }}", {});
+      }).to.throw(ExprParseError);
+    });
+
+    it("raises when the type of the LHS and RHS do not match", () => {
+      expect(() => {
+        resolveExpression("boolean", "{{ params.FOO == params.BAR }}", {
+          FOO: numberV(22),
+          BAR: boolV(true),
+        });
+      }).to.throw(ExprParseError);
+    });
+
+    it("it determines whether the LHS resolves to the same thing as the RHS", () => {
+      expect(
+        resolveExpression("boolean", "{{ params.FOO == params.BAR }}", {
+          FOO: numberV(22),
+          BAR: numberV(22),
+        })
+      ).to.be.true;
+      expect(
+        resolveExpression("boolean", "{{ params.FOO == params.BAR }}", {
+          FOO: numberV(22),
+          BAR: numberV(11),
+        })
+      ).to.be.false;
+      expect(
+        resolveExpression("boolean", "{{ params.FOO == params.BAR }}", {
+          FOO: stringV("bar"),
+          BAR: stringV("bar"),
+        })
+      ).to.be.true;
+      expect(
+        resolveExpression("boolean", "{{ params.FOO == params.BAR }}", {
+          FOO: stringV("bar"),
+          BAR: stringV("baz"),
+        })
+      ).to.be.false;
+      expect(
+        resolveExpression("boolean", "{{ params.FOO == params.BAR }}", {
+          FOO: boolV(true),
+          BAR: boolV(true),
+        })
+      ).to.be.true;
+      expect(
+        resolveExpression("boolean", "{{ params.FOO == params.BAR }}", {
+          FOO: boolV(true),
+          BAR: boolV(false),
+        })
+      ).to.be.false;
+    });
+  });
+
+  describe("Ternary expressions conditioned on an equality test", () => {
+    it("raises when the LHS param does not exist", () => {
+      expect(() => {
+        resolveExpression("number", "{{ params.FOO == 22 ? 10 : 0 }}", {});
+      }).to.throw(ExprParseError);
+    });
+
+    it("raises when the type of the LHS and RHS do not match", () => {
+      expect(() => {
+        resolveExpression("number", "{{ params.FOO == 'asdf' ? 10 : 0 }}", {
+          FOO: numberV(22),
+        });
+      }).to.throw(ExprParseError);
+    });
+
+    it("raises when the ternary expression evaluates to something of the wrong type", () => {
+      expect(() => {
+        resolveExpression("boolean", "{{ params.FOO == 22? 10 : 0 }}", {
+          FOO: numberV(22),
+        });
+      }).to.throw(ExprParseError);
+    });
+
+    it("raises when the ternary expression evaluates to a missing parameter", () => {
+      expect(() => {
+        resolveExpression("number", "{{ params.FOO == 22 ? params.BAZ : params.QUOZ }}", {
+          FOO: numberV(22),
+        });
+      }).to.throw(ExprParseError);
+      expect(() => {
+        resolveExpression("number", "{{ params.FOO == 22 ? params.BAZ : params.QUOZ }}", {
+          FOO: numberV(11),
+        });
+      }).to.throw(ExprParseError);
+    });
+
+    it("it provides resolved parameters when the ternary expression calls for them", () => {
```

Comment:
Looking at these, I wish `>, >=, <, <=` were also supported, but I also don't know if there are usecase for it 🤷🏼

Reply:
Keep Tuned (TM)